.github/workflows/storybook.yml

```diff
@@ -19,42 +19,19 @@ jobs:
       -
         name: Checkout
         uses: actions/checkout@v4
-      -
-        name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-      -
-        name: Build Docker images
-        uses: docker/bake-action@v4
-        with:
-          pull: true
-          load: true
-          files: |
-            compose.yaml
-          set: |
-            *.cache-from=type=gha,scope=${{github.ref}}
-            *.cache-from=type=gha,scope=refs/heads/main
-            *.cache-to=type=gha,scope=${{github.ref}},mode=max
       -
         name: Start services
-        run: docker compose -f compose.yaml -f compose.ci.yaml up --wait --no-build
-      -
-        name: Check HTTP reachability
-        run: curl -v --fail-with-body http://localhost
-      -
-        name: Check PWA reachability
-        run: "curl -vk --fail-with-body -H 'Accept: text/html' https://localhost"
-      -
-        name: Check API reachability
-        run: "curl -vk --fail-with-body -H 'Accept: application/ld+json' https://localhost"
+        run: docker compose -f compose.yaml -f compose.ci.yaml up --wait --build || (docker compose -f compose.yaml -f compose.ci.yaml logs pwa && exit 1)
```

Comment:
FTR: I added the last part of this line (`|| (docker compose -f compose.yaml -f compose.ci.yaml logs pwa && exit 1)`) for debug purposes, but now that the CI job works, maybe we can get rid of it